GNUmakefile

```
@@ -93,23 +89,25 @@ else
   export BASE_TAG
 endif
 
+# Specify the name of cstor-base image
 CSTOR_BASE_IMAGE= openebs/cstor-base:${BASE_TAG}
+export CSTOR_BASE_IMAGE
 
+ifeq (${CSTOR_BASE_IMAGE_ARM64}, )
+  CSTOR_BASE_IMAGE_ARM64= openebs/cstor-base-arm64:${BASE_TAG}
+  export CSTOR_BASE_IMAGE_ARM64
+endif
+
+# Specify the name of base image for ARM64
 ifeq (${BASE_DOCKER_IMAGEARM64}, )
```

Can you refactor this one? 
```suggestion
ifeq (${BASE_DOCKER_IMAGE_ARM64}, )
```